### DISCUSSION TITLE
Is Tycho able to build older RCP application?

### DISCUSSION THREAD

**Discussion by dirkey** [eclipse-tycho/tycho#278] [Q&A] [answered]
I get an old RCP application to maintain ;-). The first thing was to establish a ci build. It is possible to compile the plugins, but product export is not working. (requires 'org.eclipse.equinox.p2.iu; org.eclipse.equinox.executable.feature.group 0.0.0')

I tried many things, without any success. But, is it even possible?

Hint! I got the following warning:

[WARNING] Bundle-ClassPath entry runtime_registry_compatibility.jar does not exist in ,,,\.m2\repository\p2\osgi\bundle\org.eclipse.equinox.registry\3.5.0.v20100503\org.eclipse.equinox.registry-3.5.0.v20100503.jar
[WARNING] Bundle-ClassPath entry e4-workbench.jar does not exist in ...\.m2\repository\p2\osgi\bundle\org.eclipse.ui.workbench\3.6.2.M20110210-1200\org.eclipse.ui.workbench-3.6.2.M20110210-1200.jar
[WARNING] Bundle-ClassPath entry compatibility.jar does not exist in ...\.m2\repository\p2\osgi\bundle\org.eclipse.ui.workbench\3.6.2.M20110210-1200\org.eclipse.ui.workbench-3.6.2.M20110210-1200.jar
[WARNING] Bundle-ClassPath entry icu-jse4.jar does not exist in ...\.m2\repository\p2\osgi\bundle\com.ibm.icu\4.2.1.v20100412\com.ibm.icu-4.2.1.v20100412.jar
[WARNING] Bundle-ClassPath entry icu-data.jar does not exist in ...\.m2\repository\p2\osgi\bundle\com.ibm.icu\4.2.1.v20100412\com.ibm.icu-4.2.1.v20100412.jar
[WARNING] Bundle-ClassPath entry runtime_registry_compatibility.jar does not exist in ...\.m2\repository\p2\osgi\bundle\org.eclipse.equinox.registry\3.5.0.v20100503\org.eclipse.equinox.registry-3.5.0.v20100503.jar
[WARNING] Bundle-ClassPath entry e4-workbench.jar does not exist in ...\.m2\repository\p2\osgi\bundle\org.eclipse.ui.workbench\3.6.2.M20110210-1200\org.eclipse.ui.workbench-3.6.2.M20110210-1200.jar
[WARNING] Bundle-ClassPath entry compatibility.jar does not exist in ...\.m2\repository\p2\osgi\bundle\org.eclipse.ui.workbench\3.6.2.M20110210-1200\org.eclipse.ui.workbench-3.6.2.M20110210-1200.jar

Where did tycho get this dependencies from? Not the application of course.

Thanks,
Dirk

**Comment by dirkey** [comment]
I want to add, I building some rcp application with tycho. I didn't notice this error in any of them.

**Comment by laeubi** [comment, marked as answer]
The org.eclipse.equinox.executable.feature is for the launcher, you should add the equinox.sdk.executable to your target to get it working.

**Reply by dirkey** [reply]
The feature is already in the product definition.

**Reply by laeubi** [reply]
Then you should provide better/more error details.
